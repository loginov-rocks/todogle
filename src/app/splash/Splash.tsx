import { Box, CircularProgress } from '@material-ui/core';
import * as React from 'react';

import Logo from '../../components/logo/Logo';

import styles from './Splash.module.css';

const Splash = () => (
  <div className={styles.root}>
    <Box mb={3}>
      <Logo />
    </Box>
    <CircularProgress />
  </div>
);

export default Splash;
