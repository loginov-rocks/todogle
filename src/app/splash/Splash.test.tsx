import { shallow } from 'enzyme';
import * as React from 'react';

import Splash from './Splash';

it('matches snapshot', () => {
  const wrapper = shallow(<Splash />);

  expect(wrapper).toMatchSnapshot();
});
