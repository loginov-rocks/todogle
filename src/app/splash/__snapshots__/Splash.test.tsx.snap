// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="root"
>
  <Styled(MuiBox)
    mb={3}
  >
    <Logo />
  </Styled(MuiBox)>
  <WithStyles(ForwardRef(CircularProgress)) />
</div>
`;
